README.md

```diff
@@ -1,55 +1,19 @@
 # WordyWorld
+
 :earth_americas: 2D mobile video game as a React Native app :iphone:
 
 ## Targeted Platforms
 
-|Platform|Supported|Notes|
-|-|-|-|
-|iOS|:white_check_mark:|Will be the first and only targeted platform at the beginning|
-|Android|:white_check_mark:|Will come once iOS version is good enough|
-|Desktop Web|:white_check_mark:|Would be nice to have but I'd rather focus on mobile versions|
-|Windows Phone|:x:|React Native's Documentation is mostly focused on Android and iOS, Windows Phone has been [discontinued](https://www.cnet.com/news/windows-10-mobile-features-hardware-death-sentence-microsoft/)|
+| Platform      | Supported          | Notes                                                                                                                                                                                             |
+| ------------- | ------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| iOS           | :white_check_mark: | Will be the first and only targeted platform at the beginning                                                                                                                                     |
+| Android       | :white_check_mark: | Will come once iOS version is good enough                                                                                                                                                         |
+| Desktop Web   | :white_check_mark: | Would be nice to have but I'd rather focus on mobile versions                                                                                                                                     |
+| Windows Phone | :x:                | React Native's Documentation is mostly focused on Android and iOS, Windows Phone has been [discontinued](https://www.cnet.com/news/windows-10-mobile-features-hardware-death-sentence-microsoft/) |
```

Comment:
Those are just `prettier` changes which is a Visual Studio Code extension for auto-formatting code